.travis.yml

```diff
@@ -33,7 +33,8 @@ env:
 matrix:
   include:
     - compiler: gcc
-      env: TASK="test"
+      env: TASK="test-core"
+    - env: TASK="test-extended"
```

Comment:
You may need to set the compiler for this task as well. Otherwise you might end up with `["gcc", "clang"]` set as compiler, I think?